dev/buildtool/__init__.py

```diff
@@ -15,6 +15,7 @@
     "orca",
     "rosco",
 ]
+SPINNAKER_RUNNABLE_NON_CORE_REPOSITORY_NAMES = ["keel"]
```

Comment:
Halyard doesn't explicitly support `keel` so it hasn't been added to the list above.

However we still want to create tags and release-* branches.
We can't easily add it to the changelog though without adding it to the BOM which requires some Halyard validation.